src/cmd/cli/command/compose.go

```diff
@@ -107,6 +131,8 @@ func makeComposeUpCmd() *cobra.Command {
 					if errors.As(context.Cause(tailCtx), &errDeploymentFailed) {
 						term.Warn(errDeploymentFailed)
 						failedServices = []string{errDeploymentFailed.Service}
+					} else if len(managedResources) > 0 {
+						term.Warnf("Defang cannot monitor status of the following managed service(s): %v.\n   Check if the managed service is up by checking your dependant service's status.", managedResources)
```

Comment:
Should this one say about how the deployment may or may not have finished?
```suggestion
						term.Warnf("Deployment may not have finished. Defang cannot monitor status of the following managed service(s): %v.\n   To check if the managed service is up, check the status of the service which depends on it.", managedResources)
```

Reply:
Actually, reading this again. I think it is guaranteed that the message on line 69 has already been printed to the console at this point. So we probably could avoid repeating ourselves and instead draw attention to the more specific issue of pointing out that the deployment may or may not have partially completed.

```suggestion
						term.Warnf("Managed services may have been deployed, but deployment of all services did not complete.")
```

Reply:
I reworded it a bit.

```suggestion
						term.Warn("Managed services have been deployed, but not all services may be available yet.")
```